A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IdentPage.js

```diff
@@ -25,17 +92,17 @@ class CDUIdentPage {
         mcdu.setTemplate([
             ["A320-200"],
```

Comment:
Maybe we should add a short comment here that this is correct and does not include the engine type, after it seems like this has already been discussed and researched at least twice with multiple people involved.

Reply:
what do you mean by engine type

Reply:
-253X for example, rather than just -200.

Reply:
so write a note stating that we are 900% sure that it is -200